src/commands/watch/filters.rs

```diff
@@ -137,6 +137,18 @@ fn create_source_globs(cid: &str, c: &v2::Component) -> Option<Vec<String>> {
     }
 }
 
+fn concatenate_glob_friendly(workdir: &str, watch_glob: &str) -> String {
+    // Using Path::join on Windows correctly joins with a backslash. But the watchexec glob
```

Comment:
Nit: I'd rather information like this live as a doc comment on the function itself. That makes things like hover in IDEs work so that readers are forced to always come to the function definition should they want to learn more.